src/app/pages/ArticlePage/Byline/fixture/index.js

```diff
@@ -2,6 +2,8 @@ export const bylineWithNoRole = [
   {
     type: 'contributor',
     model: {
+      topicId: '',
+      topicUrl: '/news/topics/c8qx38nq177t',
```

Comment:
Is this where the `topicId` and `topicUrl` values will sit within the block? Do we have any data from Test Optimo or a set of example data of how it will look? Just imagined this to be more nested than sitting at the top level.

Reply:
Yep, Jim shared this with me: https://jira.dev.bbc.co.uk/secure/attachment/1106690/defaultAfterAres.json
It looks like it's going to sit within the top level of the `contributor` block.

Reply:
Ah nice! Thanks for linking that 👍🏼